### DISCUSSION TITLE
Сайт перенесён на новый адрес https://svodd.ru

### DISCUSSION THREAD

**Discussion by audetv** [terratensor/svodd#145] [Объявления]
Произведен перенос сайта на новый домен https://svodd.ru/
Ведутся работы по настройке пересылки со старого домена fct-search.audetv.ru на новый домен svodd.ru
Спасибо за понимание.

**Comment by audetv** [comment]
Товарищи, поздравляю. Переезд сайта успешно завершён.
Пересылка настроена. Все старые ссылки работают и ведут на новый адрес svodd.ru

**Comment by iprst** [comment]
Поздравляю! Ура!
